### DISCUSSION TITLE
Deeplinking/universal links.

### DISCUSSION THREAD

**Discussion by camrun91** [ApollosProject/apollos-templates#426] [Q&A] [answered]
I am trying to get linking set up to function in our app. Is there a guide on what is handled by core and what we need to do on our own to get this set up and working? @redreceipt or @conrad-vanl

**Comment by redreceipt** [comment, marked as answer]
this is a good starting point, likely we need to add to these docs:

https://github.com/ApollosProject/apollos-templates/blob/master/UNIVERSAL_LINKS.md

Essentially the server is already set up for it and the app template has the server URL listed in the associated domains. what you want is to ping the /.well-known/apple-app-site-association endpoint and get back the JSON that Apple needs. Filling out the variables in the doc should get that for you.

**Comment by redreceipt** [comment]
@camrun91 is this resolved?

**Reply by camrun91** [reply]
Yes, basically we needed to change the applink to point to our API.
